control/tests/lti_test.py

```diff
@@ -250,3 +250,14 @@ def test_squeeze_exceptions(self, fcn):
             sys.frequency_response([[0.1, 1], [1, 10]])
             sys([[0.1, 1], [1, 10]])
             evalfr(sys, [[0.1, 1], [1, 10]])
+
+        with pytest.raises(DeprecationWarning, match="LTI `inputs`"):
+            assert sys.inputs == sys.ninputs
```

Comment:
The assert .. = part is never evaluated after sys.ninputs throws the exception.

Reply:
Sorry, was working on multiple changes and once and pushed the wrong version.  New one was what was intended.